rust/osm2lanes/src/transform/tags_to_lanes/mod.rs

```diff
@@ -255,3 +183,91 @@ fn assemble_ltr(
         }
     })
 }
+
+pub fn unsupported(tags: &Tags, _locale: &Locale, warnings: &mut RoadWarnings) -> ModeResult {
+    if tags.is("highway", "bus_guideway") {
+        return Err(RoadMsg::unimplemented_tag("highway", "bus_guideway").into());
+    }
+
+    if tags.is("highway", "construction") {
+        return Err(RoadMsg::unimplemented_tag("highway", "construction").into());
+    }
+
+    let tag_tree = tags.tree();
+    if tag_tree.get("lanes").is_some() {
+        warnings.push(
+            RoadMsg::Unimplemented {
+                description: Some("lanes=*".to_owned()),
+                // TODO, TagTree should support subset
+                tags: Some(tags.subset(&["lanes"])),
+            }
+            .into(),
+        );
+    }
+
+    const ACCESS_KEYS: [&'static str; 43] = [
```

Comment:
Could be helpful to document a source for this kind of info, like https://taginfo.openstreetmap.org/keys/access#values